### PR TITLE
ICD10 scripts fix 2021 dec

### PR DESCRIPTION
**Pull request by irinazherko**
For ICD10, ICD10CM, ICD10GM:
- create current date backup of concept_relationship_manual table
- fix of the script, which deprecates previous inaccurate mapping and inserts new mapping
- addition the new target vocabularies (cancer modifier, OMOP extension) to load_stage
- addition the full fast recreate
- fix of the readme files

### CODE REVIEW

**Review by Alexdavv — APPROVED** [review]

File: ICD10/manual_work/crm_changes.sql [hunk 1]
```diff
@@ -13,10 +13,78 @@
...
+SET invalid_reason = 'D',
+    valid_end_date = current_date
+
+--SELECT * FROM concept_relationship_manual crm --use this SELECT for QA
+WHERE invalid_reason IS NULL --deprecate only what's not yet deprecated in order to preserve the original deprecation date
+
+    AND concept_code_1 IN (SELECT icd_code FROM refresh_lookup_done) --work only with the codes presented in the manual file of the current vocabulary refresh
+
+    AND NOT EXISTS (SELECT 1 --don't deprecate mapping if the same exists in the current manual file
+                    FROM refresh_lookup_done rl
```
Comment (p-talapova) [inline ICD10/manual_work/crm_changes.sql:44]: where is this table stored?

**Review by p-talapova — APPROVED** [review]

**Review by divakhnenko — APPROVED** [review]

File: ICD10/manual_work/crm_changes.sql (hunk 1 above)
Comment (irinazherko) [reply ICD10/manual_work/crm_changes.sql:44]: refresh_lookup_done table should be created at every refresh

**Comment by Alexdavv** [comment]
@dimshitc Do we merge?

**Comment by Alexdavv** [comment]
@irinazherko There's a conflict to be resolved before merge.

**Review by dimshitc — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Pull request merged**